index.html

```diff
@@ -0,0 +1,81 @@
+<!DOCTYPE html>
+<html>
+
+<style type="text/css">
+
+	body {
+
+		background-color: #3CA2E0;     
+	}
+
+	.box {
+
+		width: 40%;
+        	height: 70%;
+        	margin: 100px auto;
+		padding-left: : 50px;

```

Comment:
properties like border, padding and margin can be merged into a single property. Please use that shorthand.